recipes/robotframework-sapguilibrary/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "robotframework-sapguilibrary" %}
+{% set version = "1.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/robotframework-sapguilibrary-{{ version }}.tar.gz
+  sha256: 4b07f209f5e8bb9588b8993ad601c4bd9150635e914a0edb0c2496ca3d66373a
+
+build:
+  skip: true  # [osx or linux or py<27]
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - pywin32 >=222
+    - robotframework >=2.9
```

Comment:
Is there some reasonable upper bound like <3?

Reply:
For python? Or RF? For robot framework there is no reasonable upper bound.

Reply:
ok.